sdk/python/kfp/components/types/type_utils.py

```diff
@@ -177,30 +179,6 @@ def get_parameter_type_field_name(type_name: Optional[str]) -> str:
         get_parameter_type(type_name))
 
 
-def get_input_artifact_type_schema(
```

Comment:
This is only used in `deprecated/`, so moved it to `deprecated/`, that way we can change `get_artifact_type_schema` without breaking `deprecated/`, which we use for sample tests.